widget_tweaks/tests.py

```diff
@@ -119,6 +119,15 @@ def test_set_data(self):
         res = render_field('simple', 'set_data', 'key:value')
         assertIn('data-key="value"', res)
 
+    def test_replace_type(self):
```

Comment:
Looks good, nice and clean test. Some redundancy in the test since the only thing that changes is `date` and `hidden` here.